src/components/DayOverview/DayOverview.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import supabase, { Schema } from "@/supabase";
+import DayOverviewButton from "@/components/DayOverview/DayOverviewButton";
+
+export type ParcelOfSpecificDateLocation = {
+    collection_datetime: Schema["parcels"]["collection_datetime"];
+    clients: {
+        flagged_for_attention: Schema["clients"]["flagged_for_attention"];
+        full_name: Schema["clients"]["full_name"];
+        address_postcode: Schema["clients"]["address_postcode"];
+        delivery_instructions: Schema["clients"]["delivery_instructions"];
+    } | null;
+};
+
+export const getParcelsOfSpecificDateLocation = async (
+    date: Date,
+    collectionCentre: string
+): Promise<ParcelOfSpecificDateLocation[]> => {
+    const startDateString = date.toISOString();
+    const endDate = new Date();
+    endDate.setDate(date.getDate() + 1);
+    const endDateString = endDate.toISOString();
+
+    const { data, error } = await supabase
+        .from("parcels")
+        .select(
+            "collection_datetime, clients(flagged_for_attention, full_name, address_postcode, delivery_instructions)"
+        )
+        .gte("collection_datetime", startDateString)
+        .lt("collection_datetime", endDateString)
+        .eq("collection_centre", collectionCentre)
+        .order("collection_datetime");
+
+    if (error) {
+        throw new Error(`Database error: ${JSON.stringify(error)}`);
+    }
+
+    return data;
+};
+
+interface Props {
+    text: string;
+}
+
+const DayOverview = async ({ text }: Props): Promise<React.ReactElement> => {
```

Comment:
can this not just be passed in as `text` not `{text}`? Would also remove the need for the props interface.

Reply:
So we talked about this in person, but on second thoughts you can defo do it as Sam mentioned below idk why I thought for a second you couldn't

Reply:
That is only valid for functions. This is a component, and props have to be passed in as an object